blueprints/test-framework-detector.js

```diff
@@ -11,14 +11,22 @@ module.exports = function(blueprint) {
     var type;
 
     var dependencies = this.project.dependencies();
-    if ('ember-qunit' in dependencies || 'ember-cli-qunit' in dependencies) {
+    if ('ember-qunit' in dependencies) {
       type = 'qunit';
 
+    } else if ('ember-cli-qunit' in dependencies) {
+      let checker = new VersionChecker({ root: this.project.root });
+      if (fs.existsSync(this.path + '/qunit-rfc-232-files') && checker.for('ember-cli-qunit', 'npm').gte('4.1.1')) {
+        type = 'qunit-rfc-232';
+      } else {
+        type = 'qunit';
+      }
+
     } else if ('ember-mocha' in dependencies) {
       type = 'mocha-0.12';
 
     } else if ('ember-cli-mocha' in dependencies) {
-      var checker = new VersionChecker({ root: this.project.root });
+      let checker = new VersionChecker({root: this.project.root});
```

Comment:
Should probably just be passing in `this.project` I think (not that this is a change you made)

Reply:
fixed